airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1135,6 +1135,8 @@ components:
           $ref: "#/components/schemas/SourceId"
         connectionConfiguration:
           $ref: "#/components/schemas/SourceConfiguration"
+        updateConfigurationSecrets:
```

Comment:
this strategy is different from the originally proposed one in the issue https://github.com/airbytehq/airbyte/issues/986. Why the change?

How does this implementation handle the case where a configuration has 2 secrets and only one is updated?